integration-libs/cdc/core/auth/services/user-authentication/cdc-user-register.service.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { inject, TestBed } from '@angular/core/testing';
+import { Store } from '@ngrx/store';
+import {
+    UserProfileConnector,
+    UserProfileService,
+    UserRegisterService
+} from 'feature-libs/user/profile/core';
+import { UserSignUp } from 'feature-libs/user/profile/root';
+import { CdcJsService } from 'integration-libs/cdc/root';
+import { AuthService } from 'projects/core/src/auth';
```

Comment:
Change AuthService, User, OCC_USER_ID_CURRENT imports to @spartacus/core

Reply:
noted, thanks